packages/react-utilities/src/hooks/useOnClickOutside.ts

```diff
@@ -15,6 +15,13 @@ export type UseOnClickOutsideOptions = {
    */
   callback: (ev: MouseEvent | TouchEvent) => void;
 
+  /**
```

Comment:
_No actions are required._

We discussed this in offline: we should agree how much our hooks are connected to the library:
- not connected: keep these options
- connected: remove `contains` (always use `elementContains()`) & `element` (grab from React context) options

Reply:
> (always use elementContains())

Not sure about this one, the previous implem required an explicit flag to use native behaviour which is not straightforward IMO